packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -214,11 +214,11 @@ export class EngineReportingAgent<TContext = any> {
         message.byteOffset,
         message.byteLength,
       );
-      gzip(messageBuffer, (err, compressed) => {
+      gzip(messageBuffer, (err, gzipResult) => {
```

Comment:
These changes were not related to this PR. They were tslint errors (`no-shadowed-variable`)